packages/resource-deployment/scripts/install.sh

```diff
@@ -111,11 +111,8 @@ az account set --subscription "$subscription"
 
 . "${0%/*}/push-secrets-to-key-vault.sh"
 
-# shellcheck disable=SC2154
-keyVaultUrl=$(az keyvault show --name "$keyVault" --resource-group "$resourceGroupName" --query "properties.vaultUri" -o tsv)
-echo "Fetched keyvault url $keyVaultUrl"
-
-. "${0%/*}/function-app-create.sh"
+"${0%/*}/function-app-create.sh" -g "web-api"
+"${0%/*}/function-app-create.sh" -g "web-workers"
```

Comment:
in install.sh, before calling deploy-rest-api.sh, you need to make sure that the variable functionAppName has the value "web-api-{uniqueId}" function-app-create sets this variable to the one it creates. in this case, it will have "web-workers-{uniqueId}"